getting_started_scenarios/redshift_workflow/SPECIFICATION.md

```diff
@@ -0,0 +1,198 @@
+Inserted: Star Trek Into Darkness (2013)
+Inserted: After Earth (2013)
+Inserted: The Great Gatsby (2013)
+Inserted: Divergent (2014)
+Inserted: We Are What We Are (2013)
+Inserted: Iron Man 3 (2013)
+Inserted: The Amazing Spider-Man 2 (2014)
+Inserted: Curse of Chucky (2013)
+Inserted: The Conjuring (2013)
+Inserted: Oldboy (2013)
+Inserted: Escape Plan (2013)
+Inserted: Elysium (2013)
+Inserted: Cloudy with a Chance of Meatballs 2 (2013)
+Inserted: RoboCop (2014)
+Inserted: Carrie (2013)
+Inserted: The Mortal Instruments: City of Bones (2013)
+Inserted: Captain America: The Winter Soldier (2014)
+Inserted: Need for Speed (2014)
+Inserted: Runner Runner (2013)
+Inserted: I Spit on Your Grave 2 (2013)
+Inserted: Battle of the Year (2013)
+Inserted: Behind the Candelabra (2013)
+Inserted: No se Aceptan Devoluciones (2013)
+Inserted: The Bling Ring (2013)
+Inserted: Furious 6 (2013)
+Inserted: Machete Kills (2013)
+Inserted: The World's End (2013)
+Inserted: Pitch Perfect (2012)
+Inserted: Epic (2013)
+Inserted: The Avengers (2012)
+Inserted: Metallica Through the Never (2013)
+Inserted: Oblivion (2013)
+Inserted: Dom Hemingway (2013)
+Inserted: The Hangover Part III (2013)
+Inserted: Despicable Me 2 (2013)
+50 records were added to the Movies table. 
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Query the Movies table by year. Enter a value between 2010-2014.
+Enter a year: 2012
+The identifier of the statement is cf006da9-48b9-4ff6-8274-83916353e67d
+...PICKED
+...FINISHED
+The statement is finished!
+The Movie title field is Pitch Perfect
+The Movie title field is The Avengers
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Now you will list your databases.
+Press Enter to continue...
+The database name is : awsdatacatalog
+The database name is : dev
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Would you like to delete the Amazon Redshift cluster? (y/n)
+y
+You selected to delete redshift-cluster-200
+Press Enter to continue...
+The status is deleting
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+This concludes the Amazon Redshift example mvp scenario.
+--------------------------------------------------------------------------------
+```
+
+## Metadata
+
+
+| action / scenario            | metadata file                | metadata key                            |
+|------------------------------|------------------------------|---------------------------------------- |
+| `deleteCluster`              | redshift_metadata.yaml       | redshift.java2.delete_cluster.main      |
```

Comment:
The metadata key should be like `redshift_CreateCluster` not the Java snippet name.

Reply:
All my Java snippets follow this format. I will put a note in the spec. [service].[language].[operation].[main].